RecoLocalCalo/HcalRecAlgos/interface/HcalDeterministicFit.h

```diff
@@ -0,0 +1,65 @@
+#ifndef HcalDeterministicFit_h
+#define HcalDeterministicFit_h 1
+
+#include <typeinfo>
+#include <vector>
+
+#include "CalibCalorimetry/HcalAlgos/interface/HcalTimeSlew.h"
+#include "RecoLocalCalo/HcalRecAlgos/interface/PedestalSub.h"
+
+#include "DataFormats/HcalDetId/interface/HcalDetId.h"
+#include "CalibCalorimetry/HcalAlgos/interface/HcalPulseShapes.h"
+#include "CalibFormats/HcalObjects/interface/HcalCoder.h"
+#include "CalibFormats/HcalObjects/interface/HcalCalibrations.h"
+
+
+class HcalDeterministicFit {
+ public:
+  enum NegStrategy {DoNothing=0, ReqPos=1, FromGreg=2};  
```

Comment:
Are the names of these enums intuitive enough?
This doesn't have to change, but still, if Greg suggests another logic, would it still be FromGreg?